### PR TITLE
New: `BinaryFortress.CloudShow` version 5.7.2.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20933)

### CODE REVIEW

File: manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+PackageIdentifier: BinaryFortress.CloudShow
+PackageVersion: 5.7.2.0
+PackageLocale: en-US
+Publisher: Binary Fortress SOftware
+PublisherUrl: https://www.binaryfortress.com
+PublisherSupportUrl: https://www.binaryfortress.com/Support
+PrivacyUrl: https://www.binaryfortress.com/Privacy
+#Author: 
+PackageName: CloudShow
```
Comment (ItzLevvie) [inline manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.locale.en-US.yaml:12]:
```suggestion
PackageName: CloudShow Launcher
```

File: manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BinaryFortress.CloudShow
+PackageVersion: 5.7.2.0
+PackageLocale: en-US
+Publisher: Binary Fortress SOftware
```
Comment (ItzLevvie) [inline manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.locale.en-US.yaml:7]:
```suggestion
Publisher: Binary Fortress Software
```

File: manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+License: Proprietary
+LicenseUrl: https://cloud.show/TermsOfService
+Copyright: Copyright (c) 2007-2021 Binary Fortress Software
+CopyrightUrl: https://cloud.show/TermsOfService
+ShortDescription: Turn any screen into a digital sign in minutes with CloudShow.
+#Description: 
+Moniker: cloudshow
+Tags:
+- signage
+- digital sign in
```
Comment (ItzLevvie) [inline manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.locale.en-US.yaml:23]:
```suggestion
- digital
- sign-in
```

File: manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
+#  Scope:
```
Comment (ItzLevvie) [inline manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.installer.yaml:15]:
```suggestion
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see review.

File: manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+PackageVersion: 5.7.2.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
```
Comment (OfficialEsco) [inline manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.installer.yaml:14]:
```suggestion
  InstallerType: inno
```

File: manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+#  Scope: 
+  InstallerUrl: https://binaryfortressdownloads.com/Download/BFSFiles/128/CloudShowLauncherSetup-5.7.2.exe
+  InstallerSha256: F07E7CBDA3AA14FC2A0C8E7987CB4582CB8AC983C4323E8EEDB4B2EC346D9D9D
+  InstallerSwitches:
+    Silent: /SUPPRESSMSGBOXES /NORESTART /VERYSILENT /SP-
+    SilentWithProgress: /SUPPRESSMSGBOXES /NORESTART /VERYSILENT /SP-
```
Comment (OfficialEsco) [inline manifests/b/BinaryFortress/CloudShow/5.7.2.0/BinaryFortress.CloudShow.installer.yaml:20]:
```suggestion
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**